datasets/my/aob_sanctions/crawler.py

```diff
@@ -0,0 +1,90 @@
+from typing import Generator, Dict, cast
+from lxml.etree import _Element
+from normality import collapse_spaces, slugify
+
+from zavod import Context, helpers as h
+
+
+def parse_table(table: _Element) -> Generator[Dict[str, str], None, None]:
+    """
+    First we find the headers by searching the th tags, then we iterate over the tr tags (i.e. rows).
+
+    Returns:
+        A generator that yields a dictionary of the table columns and values. The keys are the
+        column names and the values are the column values.
+    Raises:
+        AssertionError: If the headers don't match what we expect.
+    """
+    headers = [th.text_content().strip() for th in table.findall(".//th")]
+    for row in table.findall(".//tr")[1:]:
+        if headers is None:
+            headers = []
+            for el in row.findall("./td"):
+                headers.append(slugify(el.text_content()))
+            continue
+
+        cells = []
+        for el in row.findall("./td"):
+            cells.append(collapse_spaces(el.text_content()))
+
+        # There are some incorrect rows with a smaller
+        # number of values, we'll just ignore those
+        if len(cells) != len(headers):
+            continue
```

Comment:
nope, can't just ignore data - we'd be missing people who are banned. I'll add a fix